admin/scripting.rst

```diff
@@ -95,7 +95,9 @@ script library and afterwards saved or saved as. If this view is opened on a nod
 script is available ``$node`` variable with node object.
 
 .. note::
-   All parameters provided to script are accessible via $ARGS array.
+   
+   Custom attributes with name starting with $ can be set from NXSL and read from NXSL (or macro), but never sent to client and cannot be updated from client.

```

Comment:
It should not be here. It should be in chapter that describes everything about custom attributes https://netxms.org/documentation/adminguide/object-management.html#custom-attributes . As script is not the only one place where they can be used. 
Also please add some context about usage: they can be used when it is required to store some information about node that should not be modified by users or seen by them.